src/frontend/src/flows/register/index.ts

```diff
@@ -76,6 +77,11 @@ export const registerFlow = async <T>({
   }
   const result_ = await (async () => {
```

Comment:
it's me or the function as no return type defined?

Reply:
Yes, although the result is later destructured on lines 133 to 147 (after the `canceled` case has been handled).

Reply:
Ok